src/pages/AppConfigure.vue

```diff
@@ -637,6 +639,19 @@ export default {
           }
         }
       } else {
+        if (!store.getters.appName?.trim()) {
+          store.dispatch("updateAppNameError", true);
+          document.getElementById("app").scroll({ top: 0, behavior: "smooth" });
+          return;
+        } else {
+          store.dispatch("updateAppNameError", false);
+        }
+        if (userLimitError) {
+          document
+            .getElementById("configure-step-5")
+            .scrollIntoView({ top: 0, behavior: "smooth" });
```

Comment:
Let's extract this out into a utility function:

```js
function scrollTo(selector) {
  document.querySelector(selector).scrollIntoView({ top: 0, behavior: "smooth" })
}
```

Reply:
Yeah we can do that. I'll try and make these changes